conformance/README.md

```diff
@@ -14,64 +15,61 @@ List available commands:
 $ make
 
 build-test-runner-image        Build conformance test runner image
+cleanup-conformance-tests      Clean up conformance tests fixtures
 create-kind-cluster            Create a kind cluster
 delete-kind-cluster            Delete kind cluster
 help                           Display this help
 install-nkg                    Install NKG with provisioner on configured kind cluster
-prepare-nkg                    Build and load NKG container on configured kind cluster
+prepare-nkg-no-build           Load NKG and NGINX containers on configured kind cluster
+prepare-nkg                    Build and load NKG and NGINX containers on configured kind cluster
 run-conformance-tests          Run conformance tests
+undo-image-update              Undo the NKG image name and tag in deployment manifest
 uninstall-nkg                  Uninstall NKG on configured kind cluster
-update-test-kind-config        Update kind config
 ```
 ### Step 1 - Create a kind Cluster
 
 ```bash
 $ make create-kind-cluster
 ```
 
-### Step 2 - Update NKG deployment and provisioner manifests
-**Note**: this step is only required when user wants to run conformance tests using locally built image of Nginx Kubernetes Gateway
-* Set NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> to preferred values.
-* Navigate to `deploy/manifests` and update values in `deployment.yaml` as specified in below code-block.
-* Navigate to `conformance/provisioner` and update values in `provisioner.yaml` as specified in below code-block.
-* Save the changes.
-```
-.
-..
-containers:
-- image: <repo_name>:<image_tag>
-  imagePullPolicy: Never
-..
-.
-```
+### Step 2 - Build Nginx Kubernetes Gateway container and load it and the NGINX container to configured kind cluster
+```bash
+$ make NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> prepare-nkg
 
-### Step 3 - Build and load Nginx Kubernetes Gateway container to configured kind cluster
-**Note**: this step is only required when user wants to run conformance tests using locally built image of Nginx Kubernetes Gateway
+```
+**Optional** Instead of the above command, you can skip the build NKG image step by running
 
 ```bash
-$ make NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> prepare-nkg
+$ make NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> prepare-nkg-no-build
 
 ```
-### Step 4 - Build conformance test runner image
+### Step 3 - Build conformance test runner image
 ```bash
 $ make build-test-runner-image
 ```
 
-### Step 5 - Install Nginx Kubernetes Gateway
+### Step 4 - Install Nginx Kubernetes Gateway
 ```bash
-$ make install-nkg
+$ make NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> install-nkg
 ```
 
-### Step 6 - Run Gateway conformance tests
+### Step 5 - Run Gateway conformance tests
 ```bash
-$ make run-conformance-tests
+$ make NKG_PREFIX=<repo_name> NKG_TAG=<image_tag> run-conformance-tests
```

Comment:
Do we want `PREFIX` and `TAG` here instead of `NKG_PREFIX` and `NKG_TAG`?

Reply:
Yes, thank you!